jina/docker/checker.py

```diff
@@ -74,6 +76,15 @@ def get_exist_path(directory, s):
     if os.path.exists(r):
         return r
 
+def get_summary_path(image_name: str):
+    return os.path.join(tempfile.gettempdir(), image_name.replace('/', '_'), 'summary.json')
```

Comment:
This will fail to create the directory `image_name.replace('/', '_')` and hence cannot find the file to write. Fixing this (avoid joining part 2 & 3)

Reply:
oh, right, you can also do `f"image_name.replace('/', '_')-summary.json"`

Reply:
This is done.